pom.xml

```diff
@@ -215,4 +45,14 @@
             <url>https://repo.clojars.org</url>
         </repository>
     </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>codox</groupId>
+            <artifactId>codox</artifactId>
+            <version>0.10.7</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
```

Comment:
❓ Not sure what we want to do with this thing. Do we need to move it and doc generation to the child modules?